tests/integration/targets/manager_test/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: Create session
```

Comment:
Please note that this directory is named wrong. It must be named after the module it tests, i.e. `ilo_redfish_config` instead of `manager_test`. You also need an `aliases` file (with content `unsupported`).

Also you don't have tests for the `_info` module right now.

Reply:
Done